src/unix/notbsd/linux/mod.rs

```diff
@@ -508,6 +508,7 @@ extern {
     pub fn unshare(flags: ::c_int) -> ::c_int;
     pub fn syscall(num: ::c_long, ...) -> ::c_long;
     pub fn sethostname(name: *const ::c_char, len: ::size_t) -> ::c_int;
+    pub fn memrchr(cx: *const ::c_void, c: ::c_int, n: ::size_t) -> *mut ::c_void;
```

Comment:
This ends up conflicting with the definition in `notbsd/mod.rs` as well, unfortunately.